src/components/ContentPreview/ContentPreview.js

```diff
@@ -219,8 +220,10 @@ class ContentPreview extends PureComponent<Props, State> {
         const { file: prevFile }: State = prevState;
         let loadPreview = false;
 
-        // Load preview if file version ID has changed
-        if (file && file.file_version && prevFile && prevFile.file_version) {
+        if (!file) {
+            loadPreview = !!this.fileError;
+            // Load preview if file version ID has changed
```

Comment:
Can you move comment inside the below block

Reply:
last if/else block `loadPreview = !prevFile;` since !!file is always true

Reply:
```js
if (!file) { loadPreview = !!this.fileError; }
else if (!prevFile) { loadPreview = true; }
else { loadPreview = file.file_version.id !== prevFile.file_version.id; }
```

I think can be simplified to that, file_version should always be on file object, right? If not the check can remain